### PR TITLE
Fix 2025 and cleanup

### PR DESCRIPTION
**Pull request by jvonau**
In place of #2027 needs clean install testing

### CODE REVIEW

**Comment by holta** [comment]
It's premature to comment out EJABBERD, OSM, PATHAGAR.

(Several people have asked me very directly not to do so.)

File: scripts/ansible-2.9.x [hunk 1]
```diff
@@ -76,7 +76,8 @@ elif [ -f /etc/debian_version ]; then    # Includes Debian, Ubuntu & Raspbian
 
     echo -e "\napt update; apt install ansible and python3 dependencies\n"
```
Comment (holta) [inline scripts/ansible-2.9.x]:
```suggestion
    echo -e "\napt update; apt install ansible and 7 python3-* dependencies explained at https://github.com/iiab/iiab/tree/master/scripts/ansible.md\n"
```

File: scripts/ansible.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+   https://github.com/ansible/ansible/issues/47736
+
+4. postgresql_db: python3-psycopg2
+   https://docs.ansible.com/ansible/latest/modules/postgresql_db_module.html#postgresql-db-module
+
+5. postgresql_dbuser: python3-psycopg2
+   https://docs.ansible.com/ansible/latest/modules/postgresql_user_module.html#postgresql-user-module
+
+6. htpasswd: python3-passlib
+   https://docs.ansible.com/ansible/latest/modules/htpasswd_module.html?highlight=htpasswd
```
Comment (holta) [inline scripts/ansible.md:24]: Please also explain the last one(s):

> "virtualenv # (pulls in python3-distutils python3-virtualenv)"

Possibly a quickie link to https://github.com/iiab/iiab/issues/1714 is enough?

**Review by holta — COMMENTED** [review]
Advertise ansible.md (with explanation of all 7 python3-* packages) so folks find it.

File: scripts/ansible-2.9.x (hunk 1 above)
Comment (holta) [reply scripts/ansible-2.9.x]: Please also advertise the ansible.md URL within scripts/ansible and scripts/ansible-2.8

Comment (holta) [reply scripts/ansible-2.9.x]: Also a link from the PRO TIPS section at the bottom of http://FAQ.IIAB.IO Item 11... can't hurt:

"What is Ansible and what version should I use?"
http://wiki.laptop.org/go/IIAB/FAQ#What_is_Ansible_and_what_version_should_I_use.3F

Comment (jvonau) [reply scripts/ansible-2.9.x]: I've pushed revised wording.

File: scripts/ansible.md (hunk 2 above)
Comment (jvonau) [reply scripts/ansible.md:24]: pushed revised wording

**Comment by holta** [comment]
Currently smoke-testing './iiab-install --reinstall' on Ubuntu 18.04 and Raspbian Lite...so far so good in Stage 7 or 9...

Needs a more full smoke-test on a fresh OS...

**Pull request merged**